python/nav/web/templates/info/event/details.html

```diff
@@ -248,6 +248,7 @@ <h4>Available actions</h4>
   {# Acknowledge alert #}
   <div id="actions-acknowledge" class="action-container">
     <form action="{% url 'status2_acknowledge_alert' %}">
+      {% csrf_token %}
```

Comment:
I feel rusty here... is there some JavaScript magic that turns the submit of this form into a POST request rather than the default GET (which one would expect without an explicit method on the `form` element) ?